data/strings/sound.txt

```diff
@@ -36,9 +37,17 @@
     tr = Sağdan devam edin.
     uk = Тримайтеся правіше.
     vi = Đi về phía phải.
-    zh-Hans = 向右走。
+    zh-Hans = 向右行驶
     zh-Hant = 向右走。
 
+  [make_a_slight_right_turn_street]
+    comment = Optionally used inside dist_direction_onto_street in position 3 when the normal make_a_slight_right_turn is insufficient.
+    nl = de flauwe bocht naar rechts
```

Comment:
Can you please remind me what happens if there is no en and other translations? Will an empty string be returned for all languages except nl?

And if there is a `en` translation, then it will be returned for any language, right?

Reply:
Correct, adding an `en` here would require that all languages *not* having a unique string for "make a slight right turn onto $name" (different from english anyway) would need to be specified as blank here.